build-system/test-configs/conformance-config.textproto

```diff
@@ -17,12 +17,6 @@ requirement: {
   allowlist: 'src/layout.js'
 }
 
-requirement: {
-  type: BANNED_PROPERTY_CALL
-  error_message: 'toggleAttribute is not supported in IE and iOS prior to 13.3. Switch to setAttribute and removeAttribute.'
-  value: 'Element.prototype.toggleAttribute'
```

Comment:
Let's not remove this altogether until the polyfill is in place and instead just add `allowlist: 'src/dom.js'`